Tests/Fixtures/UrlLoadBalancerFixtures.swift

```diff
@@ -30,17 +29,10 @@ extension UrlLoadBalancerFixtures {
     struct Blockchain: ServiceFixture {
         let serviceName = "Blockchain"
```

Comment:
where are these strings used ?

Reply:
Ah, yes... they were vestigial strings from the original code - will remove - thanks!

Reply:
Actually, they're used to provide contextual information for the testing logic in `func testUrlRotation(fixture: ServiceFixture, with transProto: TransportProtocol)`... but this did help me catch an error where I had the URL type set to "Consensus" in that code, instead of setting it to the value for the particular service fixture (the definitions you saw).